src/makeRobotAccountant.js

```diff
@@ -6,7 +7,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 3;
+
+  return (firstNum) => {
+    return (secondNum) => {
+      counter--;
+
+      const condition = counter < 0 && counter % 2 !== 0;
```

Comment:
it works fine now, right, but it's not clear what `condition` means, condition to render error? seems a bit confusing, try to invert this condition or rename it so it'll be clear what will happen if it's true
![image](https://user-images.githubusercontent.com/37161315/174760316-16ddff3c-04ca-41ed-8d84-e98157983dcc.png)